x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/translations.ts

```diff
@@ -62,3 +62,10 @@ export const ENABLEMENT_DESCRIPTION_BOTH = i18n.translate(
       'Your entity store is currently empty. Add information about your entities directly from your logs, or import them using a text file.',
   }
 );
+
+export const ENABLEMENT_WARNING_SELECT_TO_PROCEED = i18n.translate(
+  'xpack.securitySolution.entityAnalytics.entityStore.enablement.description.both',
```

Comment:
Not 100% on this being the correct id. Is the end part "both" correct usage?

Reply:
The end looks weird to me. I looked into [this doc](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md). It says, "Each message ID should end with a type. For example..." 
It probably should end with something like: "enablementWarningMessage"

Reply:
Aw amazing thank you! Will take a look at the docco!